wrappers/internal/service_unit_gen_test.go

```diff
@@ -17,34 +17,35 @@
  *
  */
 
-package wrappers_test
+package internal_test
 
 import (
 	"fmt"
-	"os"
-	"os/exec"
 	"strings"
+	"testing"
 	"time"
 
 	. "gopkg.in/check.v1"
 
 	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/gadget/quantity"
+	_ "github.com/snapcore/snapd/interfaces/builtin"
```

Comment:
As far as I can tell this does not seem to serve any purpose.

Reply:
This is a good observation, but actually it does serve a purpose. We need to import the package to allow init() to run from the builtin packages in order to use the interface functions, it's a common pattern in our code